infrastructure/src/main/java/org/corfudb/infrastructure/management/ReconfigurationEventHandler.java

```diff
@@ -102,4 +116,32 @@ public static boolean handleHealing(@Nonnull CorfuRuntime runtime,
             return false;
         }
     }
+
+    /**
+     * Launches a workflow to restore redundancy and merge all segments.
```

Comment:
Need more detailed comments. One liners are not sufficient.

Reply:
Done.